### ISSUE TITLE
Powah "any tier" quests show disabled starter tier variants

### ISSUE THREAD

**Issue by Quezler** [NillerMedDild/Enigmatica6#3682]
### Modpack Version

0.5.20

### Describe your issue.

The [powah quests](https://github.com/NillerMedDild/Enigmatica6/blob/master/config/ftbquests/quests/chapters/powah.snbt) use `itemfilters:tag` to check if you made anything of a certain tier, but we have disabled all the starter tier powah blocks, but they still show up as valid items to complete the quest with: https://github.com/NillerMedDild/Enigmatica6/blob/3afbc7524644056b03d12bf5d4be01eebbad0d11/kubejs/client_scripts/constants.js#L351

ideally this would be fixed by the itemfilters mod by not showing items with a certain tag and/or are hidden somehow.

dunno if itemfilters supports "nesting" items (e.g. like create filters can) in order to include everything with the tag but to also deny all of the starters.

as a "last resort" it can be turned into an `itemfilters:or` like the mekanism machines, but then you lose the magic of tags (not like any new powah tiers would get added any time soon, but still its a bit messier)

### Have you modified the modpack?

Yes

### User Modifications

FTB Library with fixed hats :3

### Did the issue happen in singleplayer or on a server?

Server

**Comment by joklbash** [comment]
This change seems to break the ability to craft a basic version of about everything in Powah. That tag is needed so the other scripts can change the crafting recipes.;

**Comment by Quezler** [comment]
try crafting the basic tier instead of starter?

**Comment by joklbash** [comment]
Yes, that's the problem - unable to craft "basic". Starter was removed from the pack a while back I think. It's removed by a script that looks for starter in the name I believe. So recipes that previously required "starter" get changed to a new recipe. 

With this change, starter no longer has the tag, so those scripts don't modify the basic recipe from what I can. So "basic" recipes are still tiered, requiring the starter item.

BTW, I'm testing this against the full dev changes, not alone against the last release. It may work fine against the last release - didn't test that.

**Comment by MuteTiefling** [comment]
@joklbash I'm not seeing any issue here. The Basic tier uses a Dielectric Casing instead of the starter tier in most cases. What recipe are you looking at exactly?

**Comment by joklbash** [comment]
Sorry, let me start over to make sure I'm telling what needs to be told. :) Yesterday, I pulled down a copy of dev. I wanted some of the bee and other changes that haven't been promoted to master. I started doing the Powah questline and was trying to craft the Furnator and Energizing Rod, both basic/first tier crafting level. 

The recipe for the basic Furnator showed it required a starter Furnator instead of the altered recipe for a blast furnace (only one recipe available). A couple other basic recipes were like this as well (think the rod was one). I undid this change that fixes the quest turn in from showing starter items and that fixed the crafting. I think the replace_input.js that changes the recipes for Powah looks for those starter tags and without them, it doesn't alter the basic recipe. I looked at so many script files that I'm not sure which one exactly may have been doing the recipe replacement.

Just to sanity check myself, I took the three lines of change for this commit and applied to the latest release and was able to duplicate the issue. Not sure if there are dependencies on other dev changes this needs in order to work, so maybe this isn't a completely valid test. When I view the recipe for the furnator, rod, thermo, magmator and hopper, it does the same thing as the full dev set did for me.

Try to insert a pic here of the recipe JEI shows. All Powah blocks are so dark, it's hard to tell that it's a basic furnator recipe.
![image](https://user-images.githubusercontent.com/3988215/143656343-733ee27a-1ea7-4ee2-8e00-83ce0cd15dd3.png)
